src/server/game/Entities/Object/Object.cpp

```diff
@@ -1789,9 +1789,9 @@ TempSummon* Map::SummonCreature(uint32 entry, Position const& pos, SummonPropert
         return nullptr;
     }
 
-    // Set the summon to the summoner's phase
     if (summoner)
-        PhasingHandler::InheritPhaseShift(summon, summoner);
+        if (summoner->IsPlayer() || !(properties && properties->GetFlags().HasFlag(SummonPropertiesFlags::IgnoreSummonerPhase)))
```

Comment:
Why is player getting a special case?

Reply:
The full name is "Ignore Summoner's Phase (Wild Only)" and I thought it meant it only affected summoners that are creatures and not players.

Is it better if I remove that part?

Reply:
"Wild" is a type of summon